ui/src/app/pages/workspace/workspace-edit.spec.tsx

```diff
@@ -244,6 +244,43 @@ describe('WorkspaceEdit', () => {
     ).toEqual(true);
   });
 
+  it('should initialize cdr versions dropdown options to the default tier when creating workspaces', async () => {
+    cdrVersionStore.set(cdrVersionTiersResponse);
+    workspace.accessTierShortName = AccessTierShortNames.Controlled;
+
+    workspaceEditMode = WorkspaceEditMode.Create;
+
+    const wrapper = component();
+    await waitOneTickAndUpdate(wrapper);
```

Comment:
I have a hope of transitioning/supplementing our Enzyme tests with React Testing Library (RTL). One of the things that I like about RTL is that you can write tests that wait for something to visually happen on the screen (an element appear, disappearing, or changing). This sort of functionality allows us to avoid using things like waitOneTickAndUpdate.

I recently tried to replicate some of RTL's functionality with a utility function called [waitForSelectorMissing](https://github.com/all-of-us/workbench/blob/eric/pact-intro/ui/src/testing/react-test-helpers.tsx#L129). This is still in a branch of mine.

I do not know if a functionality like this would be relevant here, but  either way, I thought that this might be something that you would be interested in.

Reply:
> I have a hope of transitioning/supplementing our Enzyme tests with React Testing Library (RTL).

Same! It seems like Enzyme has lost maintenance support, and that RTL has won out as the standard testing library. I've been waiting until we solidify our contract testing approach to really dive into this though.

I'm going to leave this as-is for now to keep this PR as straightforward as possible. However, I'll watch out for that function in the future since I think something like that would be useful and possibly make a transition to RTL easier.